platform/lambda/lambda.go

```diff
@@ -663,6 +664,10 @@ func isCreatingRole(err error) bool {
 func toEnv(env config.Environment, stage string) *lambda.Environment {
 	m := aws.StringMap(env)
 	m["UP_STAGE"] = &stage
+	valuesFromDotEnv, _ := godotenv.Read()
```

Comment:
We shouldn't ignore this error. The next line could panic as result.

Reply:
Thanks for pointing it, and it is fixed now.